packages/botonic-plugin-ner/src/model/model-info.ts

```diff
@@ -11,7 +12,10 @@ export class ModelInfo {
 
   constructor(readonly locale: Locale, readonly uri: string) {
     this.config = axios({ url: `${this.uri}/${CONFIG_FILENAME}` })
-    this.model = loadLayersModel(`${this.uri}/${MODEL_FILENAME}`)
+    // fetchFunc is defined to replace window.fetch to work in node environment
+    this.model = loadLayersModel(`${this.uri}/${MODEL_FILENAME}`, {
+      fetchFunc: fetch,
```

Comment:
comments must be as close as possible to the relevant line to avoid they get lost when there are refactors in the future
I suggest:
`fetchFunc: fetch, // default fetch only works in browser`